cmd/kaf/kaf.go

```diff
@@ -20,6 +22,23 @@ func getConfig() (saramaConfig *sarama.Config) {
 
 	if cluster := currentCluster; cluster.SecurityProtocol == "SASL_SSL" {
 		saramaConfig.Net.TLS.Enable = true
+		if cluster.TLS != nil && cluster.TLS.Cafile != "" {
+			caCert, err := ioutil.ReadFile(cluster.TLS.Cafile)
+			if err != nil {
+				fmt.Println(err)
+				os.Exit(1)
+			}
+			caCertPool := x509.NewCertPool()
+			caCertPool.AppendCertsFromPEM(caCert)
+
+			tlsConfig := &tls.Config{
+				RootCAs:            caCertPool,
+				InsecureSkipVerify: cluster.TLS.Verify,
+			}
+			saramaConfig.Net.TLS.Config = tlsConfig
+		} else {
+			saramaConfig.Net.TLS.Config = &tls.Config{ InsecureSkipVerify:true }
```

Comment:
i think this changes the current behavior. currently, the system ca certs are used. in this case, this turns off security and all certs will be accepted, which is a security hole.
so this else block should be removed. we could add another config option for insecureSkipVerify, is a user really wants to do it

Reply:
fixed in 1fd8a80